downstream/modules/aap-hardening/ref-apply-defensive-security-controls.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+// downstream/assemblies/assembly-hardening-aap.adoc
+
+[id="ref-apply-defensive-security-controls_{context}"]
+
+= Applying defensive security controls
+
+[role="_abstract"]
+
+The following set of defensive security controls are recommended to reduce operational risk of {PlatformNameShort} deployed on physical or virtual machines.
+These controls apply to all of the {PlatformNameShort} infrastructure nodes: the dedicated installation host, and any servers listed in the installation inventory such as controllers, hubs, EDA controllers, and the database server.
```

Comment:
for "controllers, hubs, and EDA controllers" I would recommend checking with branding to get the name/shortname of each of these and probably parameterize them similar to PlatformName/PlatformNameShort. Maybe something to do in another PR?